_data/tagging-status.yml

```diff
@@ -1533,11 +1533,11 @@
 
  - name: fixme
    type: package
-   status: unknown
+   status: partially-compatible
```

Comment:
looking at the pdf structure generated I think this can (for now) be classified as compatible

Reply:
stats updated to 'compatible'